### ISSUE TITLE
Feature request for gpg2 support

### ISSUE THREAD

**Issue by iaean** [funtoo/keychain#83]
There is an issue in Ubuntu 16.04, that results in a lot of hassle running ```keychain``` with ```gpg-agent``` at least on WSL.

The reason is, that Ubuntu 16.04 installs ```gpg-agent``` version 2.1.x and ```gpg``` client version 1.4.x. Both versions are not compatible. Distros provide ```gpg2``` packages for version 2.x. ```Keychain``` isn't aware of ```gpg2``` and Ubuntu 16.04 mix both versions. Buhh...

To overcome the situation we need ```gpg2``` support in ```keychain```.
Maybe it's possible to implement this as ```--gpg2``` flag?

### Initial error

```
 * Adding N gpg key(s): ID ...
 * Error: Problem adding (is pinentry installed?); giving up
```

### Workaround

I provide a [quick and dirty hack](/iaean/keychain) in the meantime...

Thanks for your patience.
I'm always excited to hear from you.
Best regards

**Comment by danielrobbins** [comment]
This issue has been imported into the Funtoo Linux bug tracker. This helps us to ensure that your bug is prioritized and gets the attention it deserves. You are welcome to create a Funtoo account and participate in the Funtoo Linux workflow or follow-up in the GitHub issue (comments are not currently synchronized.) Follow the action here: https://bugs.funtoo.org/browse/FL-4199

**Comment by x48rph** [comment]
Fix in version 2.8.4 via new --gpg2 flag. Thanks for report.
